docker-compose-teams.yaml

```diff
@@ -36,9 +36,27 @@ services:
       - match
       - guacamole
 
+  team-ubcthunderbots:
+    image: ubcthunderbots/tbots-software-env:0.0.1
```

Comment:
Do you have the image publicly available, so that I can have a quick look at it?

Reply:
Image is hosted here: https://hub.docker.com/r/ubcthunderbots/tbots-software-env

Let me know if you need the dockerfile itself (I think you should be able to view it on dockerhub)

Reply:
Yeah, I wanted to have a look at the Dockerfile, but the source is not shown on Docker Hub, only the raw layers.
You probably just pushed the image yourself, without connecting a GitHub Repo with the Docker Hub Repo.

Reply:
oh, I've found it through the connected issue above :D 

Looks fine :+1: